core/src/main/java/org/springframework/security/authorization/method/AuthorizationManagerBeforeReactiveMethodInterceptor.java

```diff
@@ -57,6 +58,8 @@ public final class AuthorizationManagerBeforeReactiveMethodInterceptor implement
 
 	private int order = AuthorizationInterceptorsOrder.FIRST.getOrder();
 
+	private final MethodAuthorizationDeniedHandler defaultHandler = new ThrowingMethodAuthorizationDeniedHandler();
```

Comment:
That is not needed because we are using `Mono.fromSupplier`, the `MonoSupplier` already catches the exception and calls `onError`, see [here](https://github.com/reactor/reactor-core/blob/main/reactor-core/src/main/java/reactor/core/publisher/MonoSupplier.java#L128-L136) and [here](https://github.com/reactor/reactor-core/blob/main/reactor-core/src/test/java/reactor/core/publisher/MonoSupplierTest.java#L75-L80).